src/geo/ui/search.js

```diff
@@ -245,7 +245,7 @@ var Search = View.extend({
   clean: function() {
     this._unbindEvents();
     this._destroySearchPin();
-    this.elder('clean');
+    Search.__super__.clean.call(this);
```

Comment:
Official docs recommends using this :point_up: approach too, see end of http://documentcloud.github.io/backbone/#Model-extend probably better than relying on internals that might change in the future.